src/panel/direction/DirectionPanel.jsx

```diff
@@ -310,13 +313,8 @@ export default class DirectionPanel extends React.Component {
         : <Panel
           className="direction-panel"
           onClose={this.onClose}
```

Comment:
The onClose doesn't work because the CloseButton managed by Panel is covered by the form element :/

Reply:
I set a z-index value to 1 on closeButton, is it ok for you ?